test.disabled/ejabberd_tests/tests/adhoc_SUITE.erl

```diff
@@ -29,7 +29,7 @@ groups() ->
     [{adhoc, [sequence], [ping]}].
 
 suite() ->
-    escalus:suite().
+    [{timetrap,{seconds,30}}|escalus:suite()].
```

Comment:
Timetraps are used inside the hook module.
If we stuck inside a hook module, than we would have **very** strange behaviour (most likely the travis task would be timeout).
We can use a default timetrap, but than we still need to change the default trap for bosh and mam modules.